test/std-test.cc

```diff
@@ -399,4 +399,7 @@ TEST(std_test, format_shared_ptr) {
 TEST(std_test, format_reference_wrapper) {
   int num = 35;
   EXPECT_EQ("35", fmt::to_string(std::cref(num)));
+
+  std::atomic<bool> b(true);
+  EXPECT_EQ(fmt::format("{}", std::cref(b)), "true");
```

Comment:
How is this different from the above test?

Reply:
Removed.